### ISSUE TITLE
Export/Import of the user SMTP, IMAP etc, data 

### ISSUE THREAD

**Issue by thecuki** [cypht-org/cypht#470]
## 🚀 Feature

It would be great if there is a feature that can export  servers data for one user, (then if needed edit it externally or internally with search and replace function), and then import that data to other user on the same cypht database.
The edit function is not important as long as user data can be downloaded unencrypted and manipulated on local computers, and then import back also unencrypted.

**Comment by marclaporte** [comment]
@thecuki @dumblob Are you aware of a standard format for this?

I would love to export to a format which can be imported in https://www.claws-mail.org/  But others will want other tools, so maybe we can use (or propose) a standard format?

**Comment by dumblob** [comment]
Before discussing formats, etc. I would like to clarify the motivation in form of listing well-defined use cases this should cover.

So far this sounds to me more like a one-off action which is usally better served by producing a nicely human-readable PDF with copyable text (not the character soup when copying from some PDFs).

**Comment by marclaporte** [comment]
Use cases:
* Easily share email account access between team members (ex.: info@example.org) in Cypht or other email clients (webmail, mobile or desktop)
* Easily maintaining email access in a desktop client for offline access.
* Access to IMAP info for backups. Ex.: https://imapsync.lamiral.info/

**Comment by dumblob** [comment]
Yep, these are pretty much the use cases where I would prefer a PDF (i.e. Cypht will show a very simple page with all this sensitive information revealed together with a precise time stamp and identification of the user who requested to view this page /probably the logged-in user/ and JS timer to go to Cypht previous page after a short timeout /perhaps 30 sec/ - this page can then easily be printed either on a printer or to PDF).

Basically this is only about credentials together with a very few settings (server IP/DNS, port numbers). The industry IMHO tries to not a have a standard for this as it would encourage one to use competitor's products/services.

In other words we are talking credentials management business here and not email business here :wink:. My strong preference is Bitwarden (or its open source counterpart [Vaultwarden](https://github.com/dani-garcia/vaultwarden/ )).

**Comment by marclaporte** [comment]
https://wiki.mozilla.org/Thunderbird:Autoconfiguration:ConfigFileFormat

**Comment by jonocodes** [comment]
I love this idea and the thunderbird config looks like a good direction since its used by several open source mail clients.

Here is an example of how I use thunderbird. If I'm on a dual boot system with Windows/Linux I can have my thunderbird profile stored in a shared location. Then if I install thunderbird on both sides they read the same config and have the same files. Ideally cypht would have the same experience.

But even cooler would be bringing over your thunderbird config into cypht in order to get the same mailboxes.

I think this is particularly useful for cypht which potentially has users with many mailboxes - since unified mailbox is a selling point.

**Comment by marclaporte** [comment]
@thecuki @dumblob @jonocodes 
FYI: https://github.com/cypht-org/cypht/pull/976

**Comment by marclaporte** [comment]
Closing this. Import tool has been done: https://github.com/cypht-org/cypht/pull/976

And discussions on "Declarative config for multiple accounts" continues here: https://github.com/cypht-org/cypht/issues/974
